style.css

```diff
@@ -750,6 +908,103 @@ form input::-webkit-input-placeholder {
   form button {
     margin: 0;
   }
+
+  .modal {
+    font-family: 'Inter', sans-serif;
+    color: #172b4d;
+    width: 100%;
+    margin: 0 auto;
+    background: #c1c7d0;
+    padding-top: 70px;
+    padding-bottom: 140px;
+    position: fixed;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    overflow: scroll;
+  }
+
+  .desktop-modal-content {
+    width: 920px;
+    background: #fff;
+    margin: 0 auto;
+    border-radius: 8px;
+  }
+
+  .modal .img {
+    display: block;
+    width: 872px;
```

Comment:
- [ ]  Good job by implementing a very good popup window for the desktop version. But, It seems that you forgot to add the style for the mobile version. Please, kindly make sure that your popup is responsive, and that it fits mobile and desktop sizes properly.
### your mobile popup:
![image](https://user-images.githubusercontent.com/71874425/162586975-0ed62a4b-c9c9-4e81-b33e-6f1970083086.png)
### Figma popup:
![image](https://user-images.githubusercontent.com/71874425/162587035-0872ef46-9b51-48d3-bfc0-29d0e964eea8.png)
Please, kindly fix this issues and then request for a re-review.